src/code/models/node.coffee

```diff
@@ -160,8 +180,11 @@ module.exports = class Node extends GraphPrimitive
       frames: @frames
     key: @key
 
+  canEditInitialValue: ->
+    @isDependent() or @isAccumulator or @isInCycle
+
   canEditValueWhileRunning: ->
-    @inLinks().length is 0 or @isAccumulator
+    @isDependent()
```

Comment:
Good eyes @knowuh. I had missed your comment, but had inverted that check anyway during a rebase.